src/Hero/temporary/some_image.svg.tsx

```diff
@@ -0,0 +1,70 @@
+// REACT
+import * as React from 'react';
+
+/* tslint:disable max-line-length */
+export default () => (
+    <svg
```

Comment:
We need some dummy images that aren't IP

Reply:
This could come from Galen or Jen, depending on their schedule. I agree